test/integration/docsify.test.js

```diff
@@ -33,6 +33,8 @@ describe('Docsify', function() {
 
             expect(hook.init).toBeInstanceOf(Function);
             expect(hook.beforeEach).toBeInstanceOf(Function);
+            expect(hook.beforeEmbed).toBeInstanceOf(Function);
+            expect(hook.afterEmbed).toBeInstanceOf(Function);
             expect(hook.afterEach).toBeInstanceOf(Function);
             expect(hook.doneEach).toBeInstanceOf(Function);
             expect(hook.mounted).toBeInstanceOf(Function);
```

Comment:
Sidenote, we need better hooks tests.

Reply:
Would you mind adding tests so that it shows tokens being used. The other tests are bad (lacking, obviously), but moving forward we should test new features at least.

Reply:
Hook tests were added in #1742 + #1770. Check out `/test/e2e/plugins.test.js`.